src/python/visclaw/data.py

```diff
@@ -621,12 +622,17 @@ def __init__(self, name, figno, fig_type, plotdata):
         self._show = True
         self.add_attribute('plotaxes_dict', {})
         self.add_attribute('type',fig_type)   # = 'each_frame' or 'each_run' or 'each_gauge'
+        self.add_attribute('use_for_kml',False)
+        self.add_attribute('kml_dpi',200)
```

Comment:
I guess I had in mind that this dpi setting would be specific to the KML output, and not applied to other figures generated for html output, for example.    

But then I am not really sure where this kwargs gets set.    What did you have in mind?